components/editor/TimesheetEntryEditor.tsx

```diff
@@ -96,16 +97,43 @@ const DayEditor = ({
                                 date={dateStr}
                                 timeCategories={timeCategories}
                             />
+                            <Link
+                                key={`updateEntryEditor-${entry.id}`}
+                                onClick={() => setInEditState(!inEditState)}
+                                style={{
+                                    marginRight: ".5rem",
+                                    fontWeight: "bold",
+                                }}
+                            >
+                                Close
```

Comment:
I think this button is unnecessary. This functionality could be attached to the `cancel` button of the form. I.e. the function on line 102 could be set as the `onCancel` of the `EditTimesheetEntryForm`.